routes/api.php

```diff
@@ -10,4 +11,16 @@
         Route::get('users', [UserController::class, 'index'])->name('users.index');
         Route::get('users/{user}', [UserController::class, 'show'])->name('users.show');
     });
+
+    // vaults
+    Route::prefix('vaults')->group(function () {
+        Route::get('', [VaultController::class, 'index'])->name('vaults.index')->middleware('abilities:read');
+        Route::post('', [VaultController::class, 'store'])->name('vaults.store')->middleware('abilities:read');
+
+        Route::prefix('{vault}')->group(function () {
+            Route::get('', [VaultController::class, 'show'])->name('vaults.show');
+            Route::put('', [VaultController::class, 'update'])->name('vaults.update');
+            Route::delete('', [VaultController::class, 'destroy'])->name('vaults.destroy');
+        });
```

Comment:
I would really like us to use the `Route::apiResource` method for these kind of routes, and it is so easy to use.

Reply:
done